smartseq2_single_sample/ss2_checker.wdl

```diff
@@ -0,0 +1,16 @@
+task SmartSeq2Checker {
+  File rna_metrics
+
+  String expected_rna_metrics_hash
+
+  command <<<
+    set -e
+    rna_metrics_hash=$(tail -n +6 ${rna_metrics} | md5sum | awk '{print $1}')
```

Comment:
Is this only checking the tail of the metrics file? and if so, is there a reason the full file is not checked?

Reply:
It's checking all but the first six lines. That's what the `+` does in `-n +6`. The first six lines are paths and timestamps which we don't expect to be the same across executions.